posts/2020-10-12-test-graalvm.md

```diff
@@ -0,0 +1,155 @@
+---
+title: How do we test Micronaut and GraalVM integration?
+date: Oct 12, 2020
+author: Iván López
+image: 2020-10-12.jpg
```

Comment:
@kroegerk you haven't added the image in this PR

Reply:
Oh, dear. I'm sorry @ilopmar. I assumed everything would stay together. Let me see if I can figure out how to add the image.

Reply:
Oh! You were referring to the featured image that appears on the index page. My apologies. I usually create that file and upload it directly to the folder.

Reply:
Here is the featured image I made for the post. It is already in the bgimages folder.

![2020-10-12](https://user-images.githubusercontent.com/33693760/95591620-25cd5b80-0a0d-11eb-8dbc-fd28e99b2189.jpg)

Reply:
I see that you've added it to `master` branch. I've merged `master` into this branch and it's now included.